contracts/script/avs/DeployGoerliAVS.s.sol

```diff
@@ -22,52 +18,77 @@ import { Script } from "forge-std/Script.sol";
  *      deploy script.
  */
 contract DeployGoerliAVS is Script {
-    uint96 public constant MIN_OPERATOR_STAKE = 1 ether;
-    uint32 public constant MAX_OPERATOR_COUNT = 10;
+    uint64 internal _omniChainId = 165;
+    address internal _create3Factory = address(uint160(uint256(keccak256("TODO"))));
```

Comment:
Why do we have `uint160(uint256(...))`? Can't we use only one conversion?

Reply:
sadly no